src/EnergyPlus/DataRootFinder.hh

```diff
@@ -56,9 +56,13 @@
 
 namespace EnergyPlus::DataRootFinder {
 
-constexpr int iSlopeNone(0);        // Undefined slope specification
-constexpr int iSlopeIncreasing(1);  // For overall increasing function F(X) between min and max points
-constexpr int iSlopeDecreasing(-1); // For overall decreasing function F(X) between min and max points
+enum class Slope
+{
+    Unassigned = -1,
+    None,       // Undefined slope specification
```

Comment:
Does `None` mean the same thing as `Unassigned` here?

Reply:
I'll remove Unassigned. Thanks for your feedback.

Reply:
I think it probably makes more sense to remove `None`, to keep `Unassigned` as part of every `class enum`. No need to thank me for the feedback. If you keep doing that, I will start thanking you for doing the work on GitHub also. 😄